zipline/_protocol.pyx

```diff
@@ -459,7 +466,25 @@ cdef class BarData:
                         frequency,
                         field
                     )[assets] for field in fields
-                })
+                }
+
+                if self._adjust_minutes:
+                    adjs = {
+                        field: self.data_portal.get_adjustments(
+                            assets,
+                            field,
+                            self._get_current_minute(),
+                            self.simulation_dt_func()
+                        )[0] for field in fields
+                    }
+
+                    df_dict = {field: df * adjs[field]
```

Comment:
Yeah, that logic is in data_portal.get_adjustments's `split_adj_factor`, which returns the inverse for volume